db/migrate/20200721204303_create_security_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSecurityEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :security_events do |t|
+      t.references :user, null: false
+      t.string :event_type, null: false
+      t.string :jti
+      t.string :issuer
+      t.timestamps
+    end
+    add_index :security_events, ["jti"], name: "index_security_events_on_jti"
```

Comment:
this was a guess that we *might* want it in the future, in case we ever want to find an event that somebody gave us?

Using `t.references` gives us an index on `user_id` so that access pattern is already taken care of